packages/runtime/README.md

```diff
@@ -120,7 +136,7 @@ loadRemote('app2/util').then((m) => m.add(1, 2, 3));
 
 ### loadShare
 
-- Type: `loadShare(pkgName: string)`
```

Comment:
The readme here can jump to https://module-federation.io/guide/basic/runtime.html

Reply:
like: https://www.npmjs.com/package/@rspack/core